rapids-cmake/find/package.cmake

```diff
@@ -35,28 +35,15 @@ tracking of these dependencies for correct export support.
                         all normal find_package options ]
                       )
 
-Invokes :cmake:command:`find_package` call and associate this with the listed
-build and install export set for correct export generation. Will propagate
-all variables set by :cmake:command:`find_package` to the callers scope.
+Invokes :cmake:command:`find_package <cmake:command:find_package>` call and
```

Comment:
Did this not link correctly before without the extra `<...>`?

Reply:
No it doesn't since it is cross site. So we need  the <> entry to tell sphinx what other docs to connect this too.

Reply:
Oh I see. I would have thought that the intersphinx mapping would be sufficient to point it to the other repo.